src/ProjectSystemSetup/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "ProjectSystemSetup": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\IDE\\devenv.exe",
```

Comment:
However, more generally, this is just a patch, and we should support launching VSIX project directly without needing a `launchSettings.json`. https://github.com/dotnet/roslyn-project-system/issues/1484